source/partials/_hero.html.erb

```diff
@@ -16,7 +16,7 @@
       <div class="hero-reveal"><h2 class="hero-description">Auditório do Senac POA</h2></div>
 
       <div class="hero-button">
-        <a class="button hero-reveal" href="#">
+        <a class="button hero-reveal"  href="https://www.eventbrite.com.br/e/insiter-porto-alegre-tickets-38661656054?ref=elink" target="_blank">
```

Comment:
There is a space here that's not needed. And maybe this URL could be a variable on `data/site.yml`, what do you think?

Reply:
Good point. I will change.

Reply:
Nice.